openmrs-keycloak-smart-auth/src/main/java/org/openmrs/contrib/keycloak/smart/auth/provider/SmartLaunchAuthenticator.java

```diff
@@ -208,12 +215,11 @@ public void action(AuthenticationFlowContext context) {
 			return;
 		}
 
-		appToken.getOtherClaims()
-				.forEach((key, value) -> {
-					if (value instanceof String) {
-						authSession.setUserSessionNote(SMART_NOTE_PREFIX + key, (String) value);
-					}
-				});
+		for (Map.Entry<String, Object> value : appToken.getOtherClaims().entrySet()) {
+			if (value.getValue() != null) {
+				authSession.setUserSessionNote(SMART_NOTE_PREFIX + value.getKey(), (String) value.getValue());
+			}
+		}
```

Comment:
I don't quite get why this was changed?

Reply:
Actually, we have changed the same logic to `map` in our last commit [here](https://github.com/openmrs/openmrs-contrib-keycloak-smart-auth/blob/5697511aac2e3da364f44051750d089ebe51d731/openmrs-keycloak-smart-auth/src/main/java/org/openmrs/contrib/keycloak/smart/auth/provider/SmartLaunchAccessAuthenticator.java#L175).